site-src/geps/gep-1424.md

```diff
@@ -0,0 +1,381 @@
+# GEP-1426: HTTPRoute Mesh Binding
+
+* Issue: [#1294](https://github.com/kubernetes-sigs/gateway-api/issues/1294)
+* Status: Implementable
+
+## Overview
+
+Similar to how `HTTPRoutes` bind to `Gateways` and mange North/South traffic flows in Gateway API’s ingress use-case, it would be natural to adopt a similar model for traffic routing concerns in service mesh deployments. The purpose of this GEP is to add a mechanism to the Gateway API spec for the purpose of associating `HTTPRoutes` to a service mesh and offering a model for service owners to manage traffic splitting configurations.
+
+## Personas
+
+This GEP uses the [roles and personas](https://gateway-api.sigs.k8s.io/concepts/security-model/#roles-and-personas) defined in the Gateway API security model.
+
+## Goals
+
+* SHOULD allow `HTTPRoute` traffic rules to be configurable for a mesh service by the application owner/producer.
+* SHOULD allow control by the cluster operator (mesh administrator) to grant permission for whether `HTTPRoute` resources in a given namespace are allowed to configure mesh traffic routing.
+* MUST only be allowed to configure "producer" traffic rules for a `Service` in the same namespace as the `HTTPRoute`.
+    * Traffic routing configuration defined in this way SHOULD be respected by ALL consumer services in all namespaces in the mesh.
+* SHOULD only be allowed to direct traffic to `backendRefs` within the same namespace when used for configuration of mesh traffic.
+    * This constraint may be revisited in the future, but is intended to help simplify an initial implementation.
+    * Attempting to specify an `HTTPBackendRef` in a different namespace should raise a status condition on the `HTTPRoute`.
+    * When an `HTTPBackendRef` is invalid, 500 status codes MUST be returned for requests that would have otherwise been routed to an invalid backend.
```

Comment:
I'd phrase this like so:
```suggestion
    * When a route rule contains a `HTTPBackendRef` that is invalid, 500 status codes MUST be returned for requests that would match that route rule.
```

This is the same behavior as with Gateway attachment, but also makes it clear that this applies to partially-valid HTTPRoutes.